ignite-base/package.json

```diff
@@ -9,6 +9,8 @@
     "newclear": "rm -rf $TMPDIR/react-* && watchman watch-del-all && rm -rf ios/build/ModuleCache/* && rm -rf node_modules/ && npm cache clean && npm i",
     "test": "mocha --opts Tests/mocha.opts Tests/ --recursive",
     "tron": "node_modules/.bin/reactotron",
+    "deploy-android": "fastlane android alpha", 
+    "deploy-ios": "node node_modules/react-native/local-cli/cli.js bundle --entry-file ./index.ios.js --platform ios --bundle-output ios/main.jsbundle bundle && fastlane ios beta", 
```

Comment:
I actually don't think you need to do the bundle command, it seems to do that for you in my experience.